### PR TITLE
Adjust the "Tested up to" values and Update Min PHP to 7.2

### LINKED ISSUE #60: Update to show that the plugin works with WooCommerce 5.1.0 and WordPress 5.7
https://github.com/nexcess/limit-orders/blob/develop/limit-orders.php#L17

### PR DESCRIPTION
**Pull request by bswatson**
This PR updates the "Tested up to" values within the plugin:

* WordPress: 5.5 => 6.1
* WooCommerce: 4.5 => 7.1

Fixes #60. Additionally, it bumps the versions being tested in the CI pipeline, including @szepeviktor's Travis cleanup work, so this PR closes #57.

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -6,20 +6,25 @@ includes:
...
-	autoload_files:
-		- %rootDir%/../../php-stubs/wordpress-stubs/wordpress-stubs.php
-		- %rootDir%/../../php-stubs/woocommerce-stubs/woocommerce-stubs.php
+	scanFiles:
+		- %currentWorkingDirectory%/vendor/php-stubs/woocommerce-stubs/woocommerce-stubs.php
 	paths:
 		- %currentWorkingDirectory%/src/
 		- %currentWorkingDirectory%/limit-orders.php
+	excludePaths:
+		- %currentWorkingDirectory%/vendor/*
```
Comment (szepeviktor) [inline phpstan.neon.dist:15]: This directory is not included, so should not be excluded :)

Comment (bswatson) [reply phpstan.neon.dist]: Removed! Thank you!

**Review by lukecav — APPROVED** [review]

**Comment by szepeviktor** [comment]
CI says

> Composer could not find a composer.json file in /home/runner/work/limit-orders/limit-orders/vendor/woocommerce/woocommerce-src-7.1

WC has stopped being a Composer package.

**Comment by bswatson** [comment]
> CI says
> 
> > Composer could not find a composer.json file in /home/runner/work/limit-orders/limit-orders/vendor/woocommerce/woocommerce-src-7.1
> 
> WC has stopped being a Composer package.

They moved to a monorepo and WooCommerce is now embedded as a plugin within. I'm trying to get this to run with as few changes as possible, but I might need to convert things over to use their style of `wp-env` based testing to succeed.

**Pull request merged**